### ISSUE TITLE
`prop.readOnly false` should not generate the `readOnly` attribute

### ISSUE THREAD

**Issue by MangelMaxime** [davedawkins/Sutil#91]
Similarly to #81, if you set `prop.readOnly false` in an input then the input has the `readonly` attribute making it read only.

```fs
Html.input [
    prop.value "Some text"
    prop.readOnly false
]
```

generates:

```html
<input _svid="27" readonly="false" >
```

but the input cannot be edited.

Work around:

```fs
Html.input [
	prop.value "Some text"
	if false then // Condition
		prop.readOnly true
]
```

**Comment by davedawkins** [comment]
I did think I'd fixed this. Let me look into it.

**Comment by davedawkins** [comment]
Fixed in Sutil 2.0.14
